client/webserver/site/src/js/locales.ts

```diff
@@ -316,7 +317,9 @@ export const enUS: Locale = {
   [ID_TRADING_TIER_UPDATED]: 'Trading Tier Updated',
   [ID_INVALID_TIER_VALUE]: 'Invalid tier value',
   [ID_INVALID_COMPS_VALUE]: 'Invalid comps value',
-  [ID_API_ERROR]: 'api error: {{ msg }}'
+  [ID_API_ERROR]: 'api error: {{ msg }}',
+  [ID_NOTIFICATION_BLOCKED_BROWSER_MSG]: 'Browser notifications are currently blocked. Please unblock this site in your browser settings to receive notifications.',
+  [ID_NOTIFICATION_BLOCKED_DESKTOP_MSG]: 'Notifications are currently blocked for DCRDEX. Please unblock DCRDEX in your system notification settings.'
```

Comment:
> Why is this neccessary? 

It's possible to request permission on macOS and I was working on that, but since we're just going to ignore requesting permission for desktop, I'll revert some of this changes.